docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,46 @@
 [id="con_devfile_{context}"]
-= Devfile
+= About devfiles
 
 [role="_abstract"]
 
-Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development and customize or extend the application development to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating manual processes.
+.What a devfile does for you
 
-Development tools consume devfiles and transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, users can create any number of identical workspaces from the same devfile. This way, different teams can use and share workspaces in various ways, including a devfile registry or together with the source code of a project.
+Devfiles are YAML files, and they simplify your development process. Devfiles capture instructions on how to set up your development environment and provide these instructions to your tools and IDE. The tools and IDE understand the instructions and support devfiles in order to take your development project from source code to a running application. By using devfiles, you can guarantee consistent patterns of behavior for applications written in different languages and using different tools. Devfiles also provide automation steps to your tools and IDE so that they can automate your current, manual processes. Using the recommended best practices from the devfile, the tools and IDE can: take in the repository hosting your application source code, build your code, run your application on your local container, and then deploy your application to cloud-native containers.
 
-A devfile includes:
+.Benefits of devfiles
 
-* Guidance about runtime images
+With devfiles, you can make workspaces composed of multiple containers. In these containers, you can create any number of identical workspaces from the same devfile. If you create multiple workspaces, you can share your devfile with different teams. By sharing a single devfile across multiple teams, you can ensure that each team has the same user experience and the same build, run, deploy behaviors.
```

Comment:
With devfiles, you can make workspaces composed of multiple containers. In these containers, you can create any number of identical workspaces from the same devfile. If you create multiple workspaces, you can share your devfile with different teams. By sharing a single devfile across multiple teams, you can ensure that each team has the same user experience and build, run, deploy behaviors.